src/AuthenticatedAPIClient/authInterface.js

```diff
@@ -112,6 +95,37 @@ export default function applyAuthInterface(httpClient, authConfig) {
 
   httpClient.refreshAccessToken = () => httpClient.post(httpClient.refreshAccessTokenEndpoint);
 
+  httpClient.refreshAccessTokenPromise = null;
+
+  httpClient.refreshAccessTokenOnce = () => {
+    if (!httpClient.isAccessTokenExpired(httpClient.getDecodedAccessToken())) {
```

Comment:
Don't make multiple calls to getDecodedAccessToken, because we've had the result change.  Also, please comment as such.